images/gcloud-bazel/Makefile

```diff
@@ -0,0 +1,30 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
Using a Makefile is not ideal to me...

We usually use a [cloudbuild.yaml](https://cloud.google.com/container-builder/docs/create-custom-build-steps) to build a container even for only one step (build from a Dockerfile). In this way we can ensure all the containers are build in a controlled environment with [Google Cloud Container Builder](https://cloud.google.com/container-builder/docs/)

Reply:
Done

The Makefile is necessary to get the tags I want, but it now uses gcb to push builds.